### PR TITLE
107519 trial to full xml creation

### PR DESCRIPTION
**Pull request by harshda15125**
- Added code changes for Json to Xml mapping for trial to full license event
- Added unit test cases
- Improved mutation score from stryker report
- Renamed unit test project
- Added Functional test cases

### CODE REVIEW

**Review by pravinvarade10801 — APPROVED** [review]
LGTM

**Review by kishorambare — COMMENTED** [review]
One minor comment pls.

File: tests/UKHO.ERPFacade.API.FunctionalTests/ERPFacadePayloadTestData/RoSPayloadTestData/LTC09_InValidLTCwithNoCorrelationID.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
```
[the remark this answers is not in the document]
Comment (MitaliSingh16) [reply tests/UKHO.ERPFacade.API.FunctionalTests/ERPFacadePayloadTestData/RoSPayloadTestData/LTC09_InValidLTCwithNoCorrelationID.json:1]: done

**Review by kishorambare — APPROVED** [review]
Looks good.

**Review by Vishal14583 — APPROVED** [review]
LGTM

**Review by jrippington — APPROVED** [review]
Looks good

**Pull request merged**